### PR TITLE
Create 'work in the open' principle

### LINKED ISSUE #97: Create Work in the open principle

### PR DESCRIPTION
**Pull request by edhamiltonHO**
Content migration from internal docs with rewrite to bring in line with 'writing a principle' standard, and flesh out main sections.

This will close #97

### CODE REVIEW

File: docs/principles/work-in-the-open.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+The [Home Office DDaT Strategy](https://www.gov.uk/government/publications/home-office-digital-data-and-technology-strategy-2024/home-office-digital-data-and-technology-strategy-2024) advocates working and developing in the open to realise the benefits of shared technology products and to deliver effectively at scale. This is aligned to [point 3 of the Technology Code of Practice](https://www.gov.uk/guidance/be-open-and-use-open-source) and [requirement 12 of the Service Standard](https://www.gov.uk/service-manual/service-standard/point-12-make-new-source-code-open).
+
+Working in the open helps to build institutional memory, shared knowledge and best practice. Sharing code in the open makes it easier for teams to discover patterns and components. This in turn enables reuse and the development of shared technology products, and reduces duplication of effort.
+
+---
+
+## Applications and Implications
+
+- Make your documentation, artefacts and code easily discoverable for other teams
+- Follow central government and Home Office guidance on open and closed source code
```
Comment (GraemeSutherlandHO) [inline docs/principles/work-in-the-open.md:31]: This needs a link, if we have one yet.

File: docs/principles/work-in-the-open.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- Build more secure applications by reducing reliance on 'security by obscurity'
```
Comment (GraemeSutherlandHO) [inline docs/principles/work-in-the-open.md:32]: link to eg.https://en.wikipedia.org/wiki/Security_through_obscurity

File: docs/principles/work-in-the-open.md (hunk 1 above)
Comment (edhamiltonHO) [reply docs/principles/work-in-the-open.md:31]: I can link to central gov guidance now, but will need to get the Home Office guidance online in SEGAS before can link to that

Comment (GraemeSutherlandHO) [reply docs/principles/work-in-the-open.md:31]: need a way of marking a TODO -- there's a link but not yet public/available sort of thing

File: docs/principles/work-in-the-open.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- Encourage review of work by wider communities, and welcome their feedback and contribution
```
Comment (robertdeniszczyc2) [inline docs/principles/work-in-the-open.md:33]:
```suggestion
- Encourage reviews of work by wider communities, and welcome feedback and contributions
```

File: docs/principles/work-in-the-open.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+layout: sub-navigation
+order: 1
+title: Work in the open
+date: git Last Modified
+tags:
+- Ways of working
+- Source management
+---
+
+{% import "_includes/macros/tags.njk" as pageTag %}
```
Comment (keithkennedyHO) [inline docs/principles/work-in-the-open.md:11]: Update macro import based on latest location of template.
```suggestion
{% import "macros/tags.njk" as pageTag %}
```

**Review by keithkennedyHO — APPROVED** [review]

**Pull request merged**